backend/apps/slack/management/commands/slack_sync_data.py

```
@@ -1,34 +1,24 @@
-"""A command to populate Slack channels and members data based on workspaces's bot tokens."""
-
 import logging
 import time
-
+import random
```

Ruff (0.11.9)

`random` imported but unused

Remove unused import: `random`

(F401)